src/pyscaffoldext/django/extension.py

```diff
@@ -77,18 +83,18 @@ def enforce_django_options(struct, opts):
 
     return struct, opts
 
+
 def fix_django_settings(struct, opts):
-    old_string = f"'{opts['project']}"
-    new_string = f"'src.{opts['project']}"
+    new_string = "src.{0}".format(opts['project'])
```

Comment:
In this case you remove the leading single quotes, is that the original intention?
I thought the leading quote was there to avoid replacing strings that were not intended to be replaced (and that was the reason I suggested the regex, so you could have single or double quotes).